LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Michael Okoh
+Copyright (c) 2019 Michael Okoh and Adebambo Oyelaja
```

Comment:
This should be reverted.

The Readme should contain

# Contributors
* [All Collaborators](https://github.com/ichtrojan/go-practice/graphs/contributors)

Reply:
or Have names included there in a list but not in the license